core/SConscript.unix

```diff
@@ -833,6 +839,7 @@ rust = env.Command(
     source='',
     action=cargo_build(), )
 env.Depends(rust, protobuf_blobs)
+env.Depends(rust, TRANSLATION_DATA)
```

Comment:
Don't we need to add the same rule to `SConscript.firmware`?

Reply:
it's unlikely to matter but yeah. added in rebase